src/components/outline/Display.tsx

```diff
@@ -1,11 +1,188 @@
-import React from "react";
-import { Card, Flex, Input, MenuList, MenuItem, InputGroup, Text, Icon, Button, Menu, Box, SimpleGrid, MenuButton, CardBody, InputRightElement } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { Card, Flex, Input, MenuList, MenuItem, InputGroup, Text, Icon, Button, Menu, Box, SimpleGrid, MenuButton, CardBody, InputRightElement, Checkbox, FormControl, FormLabel, Stack } from "@chakra-ui/react";
 import { ChevronDownIcon } from "@chakra-ui/icons";
 
+// temporary profile data
+const profiles = [
+  { id: 1, matched: true, school: "Georgia Institute of Technology", year: "3rd year", major: "Computer Science", description: "Hello, my name is bleh bleh bleh", commitmentLevel: "high",  skills: ["Java", "Python", "Node"], isJudging: true },
+  { id: 2, matched: true, school: "UGA", year: "2nd year", major: "Computational Media", description: "Hello, my name is bleh bleh bleh", commitmentLevel: "medium",  skills: ["C", "JavaScript"], isJudging: true },
+  { id: 3, matched: true, school: "Georgia Institute of Technology", year: "1st year", major: "Computer Science", description: "Hello, my name is bleh bleh bleh", commitmentLevel: "low",  skills: ["React", "Node", "Express"], isJudging: true },
+  { id: 4, matched: true, school: "USG", year: "3rd year", major: "Computer Science", description: "Hello, my name is bleh bleh bleh", commitmentLevel: "high",  skills: ["Machine Learning", "Python", "Node"], isJudging: true },
+  { id: 5, matched: true, school: "University of Maryland", year: "4th year", major: "Computer Science", description: "Hello, my name is bleh bleh bleh", commitmentLevel: "medium",  skills: ["HTML/CSS", "C#", "C++"], isJudging: true },
+  { id: 6, matched: true, school: "Emory", year: "3rd year", major: "Computer Science", description: "Hello, my name is bleh bleh bleh", commitmentLevel: "low",  skills: ["Java", "Python", "Node"], isJudging: true },
+];
```

Comment:
I know this was worked on before I defined the types for user card, but let's merge main to your branch and make sure your dummy profiies are of type `UserCardType[]`. As such, you will have to make some changes to these profiles to follow that schema, like nesting most of these fields under a `profiie` field. Once that's done, we can display these profiles instead of the single dummy user card displayed in main.